debian/changelog

```diff
@@ -86,6 +86,7 @@ snapcraft (2.0) xenial; urgency=medium
 
   [ Vamshi Balanaga ]
   * Added test for git sources in sources.py (LP: #1534411)
+  * Added test for bzr sources in sources.py (LP: #1536283)
```

Comment:
You shouldn't edit the changelog of the released version.
We could open a new section for the unreleased 2.1 version, but we are currently trying to do that automatically with git-buildpackage, so just remove this line, please.